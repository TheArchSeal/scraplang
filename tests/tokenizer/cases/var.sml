a
abc
ABC
a123
A_1
_
_123
