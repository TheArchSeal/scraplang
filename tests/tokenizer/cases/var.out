(5):1:1 a a
(5):2:1 abc abc
(5):3:1 ABC ABC
(5):4:1 a123 a123
(5):5:1 A_1 A_1
(5):6:1 _ _
(5):7:1 _123 _123
